lib/couchbase-orm/utilities/query_helper.rb

```diff
@@ -87,6 +89,13 @@ def build_match_hash(key, value)
                 matches.join(" AND ")
             end
 
+            def build_match_range(key, value)
+                matches = []
+                matches << "#{key} >= #{quote(value.begin)}"
+                matches << "#{key} <= #{quote(value.end)}"
+                matches.join(" AND ")
```

Comment:
Would it be the same as
 "#{key} >= #{quote(value.begin)} AND #{key} <= #{quote(value.end)}"

Reply:
Yes it is. I found the other way easier to read but I can change if you prefer?